src/explore-education-statistics-common/src/components/form/rhf/RHFFormFieldSelect.tsx

```diff
@@ -0,0 +1,35 @@
+import RHFFormField, {
+  RHFFormFieldComponentProps,
+} from '@common/components/form/rhf/RHFFormField';
+import FormSelect, {
+  FormSelectProps,
+} from '@common/components/form/FormSelect';
+import React from 'react';
+import { FieldValues, Path } from 'react-hook-form';
+
+export interface RHFFormFieldSelectProps<TFormValues extends FieldValues>
+  extends Omit<
+    FormSelectProps,
+    'name' | 'value' | 'inputRef' | 'id' | 'error'
+  > {
+  name: Path<TFormValues>;
+  id?: string;
+  showError?: boolean;
+}
+
+interface Values<TFormValues extends FieldValues>
+  extends Omit<FormSelectProps, 'name'> {
+  name: Path<TFormValues>;
+}
+
+type Props<TFormValues extends FieldValues> = RHFFormFieldComponentProps<
+  Values<TFormValues>,
+  TFormValues
+>;
+
+export default function RHFFormFieldSelect<TFormValues extends FieldValues>({
+  name,
+  ...props
+}: Props<TFormValues>) {
+  return <RHFFormField {...props} name={name} as={FormSelect} />;
+}
```

Comment:
It might be a good idea to add this for parity with `FormSelect`:

```ts
RHFFormFieldSelect.unordered = [] as [];
```

This is mostly to stop unnecessary re-rendering due to the `order` prop if we use a array literal.

```tsx
// Can re-render at any time due to [] for `order`
<RHFormFieldSelect order={[]} />

// Won't re-render due to `order`
<RHFormFieldSelect order={RHFormFieldSelect.unordered} />
```